.deploy.yml

```diff
@@ -1,34 +1,58 @@
 trigger:
 - v3
 - feature/*
-queue:
-  name: DocFX
-steps:
-- powershell: ./impact.ps1
-  env:
-    OP_BUILD_USER_TOKEN: $(OpBuildUserToken)
-    GITHUB_BASIC_AUTH: $(GitHubBasicAuth)
-    SYSTEM_ACCESS_TOKEN: $(System.AccessToken)
-  displayName: Impact Test
+pr:
+- v3
+- feature/*
+jobs:
+# unit & e2e test jobs
+- job: Linux Test(ubuntu-16.04)
+  pool:
+    vmImage: 'ubuntu-16.04'
+  condition: eq(variables['Build.Reason'], 'PullRequest')
+  steps:
+    - bash: ./build.sh
+- job: macOS Test(macOS-10.13)
```

Comment:
any of the build job fails the whole build fails?

Reply:
I haven't found the doc, but during my testing it happened once.